src/iuse_actor.cpp

```diff
@@ -257,7 +257,7 @@ int iuse_transform::use( player &p, item &it, bool t, const tripoint &pos ) cons
     return 0;
 }
 
-ret_val<bool> iuse_transform::can_use( const player &p, const item &, bool,
+ret_val<bool> iuse_transform::can_use( const Character &p, const item &, bool,
```

Comment:
It would be best practice to include Character here even though it compiles without.  Plus, that's one less thing IWYU will complain about next time it's run